### PR TITLE
bitrise-step-veracode-security 1.0.2

### PR DESCRIPTION
**Pull request by ryanphillipthomas**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/bitrise-step-veracode-security/1.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- macos
+- android
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/bitrise-step-veracode-security/1.0.2/step.yml:27]: Still, those dependencies are not used in script.

**Comment by tadam313** [comment]
hi @ryanphillipthomas thanks for the contribution! Could you please address @koral-- change requests and remove the unused dependency? Much appreciated 🙇

**Comment by ryanphillipthomas** [comment]
Yup I'll have another PR up with those changes

**Comment by tadam313** [comment]
@ryanphillipthomas thanks, and one last request. Can you please rename this step to `veracode-security`? In the future we'd like to enforce not having `bitrise` or `step` prefixes in the step name. We've already prepared the step directory: https://github.com/bitrise-io/bitrise-steplib/tree/master/steps/veracode-security?

**Comment by ryanphillipthomas** [comment]
Yup can do, I was curious about the name. Glad you mentioned that, I'll include it as well. The PR might go up over the weekend just FYI but I'll get it up. If you guys have any other notes feel free to keep adding here and I'll include.

**Comment by ryanphillipthomas** [comment]
https://github.com/bitrise-io/bitrise-steplib/pull/2940

**Pull request closed without merging**